corehq/apps/userreports/es/adapter.py

```diff
@@ -0,0 +1,44 @@
+from elasticsearch import NotFoundError
+from corehq.apps.userreports.util import get_table_name
+from corehq.apps.userreports.adapter import IndicatorAdapter
+from corehq.elastic import get_es_new
+
+
+class IndicatorESAdapter(IndicatorAdapter):
+
+    def __init__(self, config):
+        super(IndicatorESAdapter, self).__init__(config)
+        self.es = get_es_new()
+        self.table_name = get_table_name(config.domain, config.table_id)
+
+    def rebuild_table(self):
```

Comment:
just mentioning here that I think @dannyroberts is looking into changing the default to no longer create mappings: https://trello.com/c/f42dwYXK/3-set-auto-create-index-to-false so we might need to update something here if/when that happens.